src/test/tests/connectionManager.test.ts

```diff
@@ -81,22 +84,110 @@ describe('Connection Manager Tests', () => {
             }
         ].forEach(async testCase => {
             it('Input URL: ' + testCase.inputUrl, async () => {
-                // Check credentials not set
-                process.env[ConnectionManager.USERNAME_ENV] = process.env[ConnectionManager.PASSWORD_ENV] = process.env[ConnectionManager.URL_ENV] =
-                    '';
+                process.env[ConnectionManager.USERNAME_ENV] = process.env[ConnectionManager.PASSWORD_ENV] = process.env[
+                    ConnectionManager.ACCESS_TOKEN_ENV
+                ] = process.env[ConnectionManager.URL_ENV] = '';
+
+                // Store previous CLI home, and set to a non existing path so no credentials will be read from the CLI.
+                const previousHome: string = getCliHomeDir();
+                setCliHomeDir(path.resolve('/path/to/no/where'));
+
+                // Check credentials not set.
                 await connectionManager.populateCredentials(false);
                 assert.isFalse(connectionManager.areXrayCredentialsSet());
 
                 process.env[ConnectionManager.URL_ENV] = testCase.inputUrl;
                 process.env[ConnectionManager.USERNAME_ENV] = 'admin';
                 process.env[ConnectionManager.PASSWORD_ENV] = 'password';
+                process.env[ConnectionManager.ACCESS_TOKEN_ENV] = 'token';
```

Comment:
Could you please separate the basic-auth and access-token tests? I mean - test username and password and **then** test access token.